src/NUnitConsole/nunit3-console/nunit3-console.csproj

```diff
@@ -15,6 +14,9 @@
     <Compile Include="..\..\NUnitEngine\nunit.engine\Internal\ExceptionHelper.cs" LinkBase="Utilities" />
     <Content Include="..\..\..\nunit.ico" Link="nunit.ico" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.1'">
+    <PackageReference Include="Mono.Options" Version="5.3.0.1" />
+  </ItemGroup>
```

Comment:
I chose to add this in as a dependency instead of a copy/paste when I first started on this, months ago. I've gone off the idea a little now - will investigate copying it over, as per the netfx version.